app/assets/javascripts/leaflet.map.js

```diff
@@ -21,12 +37,21 @@ L.OSM.Map = L.Map.extend({
 
     this.baseLayers = [];
 
+    this.baseLayers.push(new L.OSM.VAntique({
+      attribution: 'Coastlines © <a href="https://www.openstreetmap.org/copyright" target="_blank">OpenStreetMap</a> contributors',
```

Comment:
Can we change attribution to <a href="/copyright" target="_blank">Copyright</a> ?